src/bootstrap_handler.rs

```diff
@@ -184,6 +194,33 @@ mod test {
         path.parent().unwrap().join(name)
     }
 
+    fn create_contacts() -> Contacts {
+        let addr = net::SocketAddrV4::new(net::Ipv4Addr::new(1,2,3,4), 8080);
```

Comment:
That's disappointing.  Maybe I'm remembering back to the C++ code.  I guess implementing such helpers would be beyond the scope of this task.  I'll add an issue for it.